cylc/flow/workflow_db_mgr.py

```diff
@@ -233,13 +233,15 @@ def process_queued_ops(self) -> None:
                     self.db_deletes_map.items()):
                 while db_deletes:
                     where_args = db_deletes.pop(0)
+                    # LOG.debug(f"DB {table_name} DEL {where_args}")
```

Comment:
Surely these should either be removed or uncommented?

Reply:
FYI `cylc play --main-loop='log db'` will write out a log file with all the SQL operations performed on the database in it.

Reply:
```suggestion
```

Reply:
It's a pity Python doesn't have multiple levels of debug logging, so we can have really fine-grained logging on demand without having too much log bloat for less serious debugging.

BTW I had initially thought this bug was causing loads of redundant method calls to update a map of DB inserts, but not actually causing redundant DB updates - hence the debug statements here - but it's a queue not a map, so it would cause both.